lxqtleave/translations/lxqt-hibernate_it.desktop

```diff
@@ -0,0 +1,15 @@
+[Desktop Entry]
+Type=Application
+Name=Hibernate
+GenericName=Hibernate
+Comment=hibernate the machine
+Exec=lxqt-leave --hibernate
+Icon=system-suspend-hibernate
+Categories=LXQt;X-Leave
+OnlyShowIn=LXQt
+
+#TRANSLATIONS_DIR=../translations
+
+# Translations
```

Comment:
There has to be an issue with this, looking at the italian desktop files
with only the #translation part they are not built correctly
http://i.imgur.com/jgCxW4I.png
The universal keys are all missing:

$ cat lxqt-shutdown_it.desktop
#Translations
Name[it]=Arresta
Comment[it]=Spegnere il computer

The  desktop files  in other languages are in source all complete
including the universal part.


Il 07/04/2015 00:15, palinek ha scritto:
> In lxqtleave/translations/lxqt-hibernate_it.desktop
> <https://github.com/lxde/lxqt-session/pull/24#discussion_r27836012>:
> 
>> @@ -0,0 +1,15 @@
>> +[Desktop Entry]
>> +Type=Application
>> +Name=Hibernate
>> +GenericName=Hibernate
>> +Comment=hibernate the machine
>> +Exec=lxqt-leave --hibernate
>> +Icon=system-suspend-hibernate
>> +Categories=LXQt;X-Leave
>> +OnlyShowIn=LXQt
>> +
>> +#TRANSLATIONS_DIR=../translations
>> +
>> +# Translations
> 
> Please, remove all universal keys and leave here only the translated
> ones ([it]).
> 
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/lxde/lxqt-session/pull/24/files#r27836012>.
>